index.js

```diff
@@ -0,0 +1,102 @@
+const playersData = "http://localhost:3000/players"
+const playerContainer = document.querySelector(".player-container")
+
+// DELIVERABLE #1
+
+// get player data
+fetch(playersData)
+    .then(response => response.json())
+    .then(data => buildPlayerElements(data))
+
+// build all players
+function buildPlayerElements(data) {
+    data.forEach(buildPlayer)
+}
+
+// build one player
+function buildPlayer(player) {
+    const div = document.createElement("div")
+    div.classList.add("player")
+    div.setAttribute("player-id", player.id)
```

Comment:
This will work, but it's better to use a dataset in cases like these - since "player-id" is not a standard HTML attribute. You can use this instead:

```js
div.setAttribute("data-id", player.id)
```

*or*

```js
div.dataset.id = player.id
```